README.md

```diff
@@ -158,39 +169,33 @@ func main() {
 
 ```
 
-## Additional resources ##########################################################
+## Contributing 
+
+Please feel free to submit issues, fork the repository and send pull requests!
+
+When submitting an issue, we ask that you please include a complete test function that demonstrates the issue.
+
+## Additional resources
 
 Additional resources are available at:
 
-* [Package documentation](https://pkg.go.dev/github.com/cloudinary/cloudinary-go)
+* [Package reference documentation](https://pkg.go.dev/github.com/cloudinary/cloudinary-go)
+* [SDK Documentation](https://cloudinary.com/documentation/go_integration)
```

Comment:
Maybe also add a bullet linking to the reference?

Reply:
The reference is linked above.

Reply:
I will also add a badge with the link to the reference on top of the document

Reply:
added